resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -0,0 +1,1877 @@
+          description: Select the platform used to sequence your samples.
+          required: false
+          choices:
+            - label: NextSeq 500
+              value: nextseq_500
+            - label: HiSeq 2500
+              value: hiseq_2500
+            - label: HiSeq 2000
+              value: hiseq_2000
+            - label: NovaSeq 6000
+              value: novaseq_6000
+            - label: Other
+              value: other
+        - name: target_of_assay
+          label: Target of assay
+          type: basic:string
+          description: |
+            Target in a ChIP-seq experiment, e.g. transcription factor, histone mark, etc.
+          required: false
+          hidden: "experiment.assay_type != 'chip-seq' && experiment.assay_type != 'chipmentation'"
+          choices:
+            - label: Transcription factor
+              value: transcription_factor
+            - label: Histone
+              value: histone
+            - label: Control
+              value: control
+            - label: Chromatin associated protein
+              value: chromatin_associated_protein
+            - label: RNA binding protein
+              value: rna_binding_protein
+            - label: Chromatin remodeller
+              value: chromatin_remodeller
+            - label: Nucleotide modification
+              value: nucleotide_modification
+            - label: Other
+              value: other
+        - name: antibody_target
+          label: Antibody target
+          type: basic:string
+          required: false
+          hidden: "experiment.assay_type != 'chip-seq' && experiment.assay_type != 'chipmentation'"
+          description: Antigen that you are enriching for, e.g. Myc, H3K27AC, etc.
+        - name: antibody_manufacturer
+          label: Antibody manufacturer
+          type: basic:string
+          required: false
+          hidden: "experiment.assay_type != 'chip-seq' && experiment.assay_type != 'chipmentation'"
+          description: Name of the vendor where the antibody used came from.
+          choices:
+            - label: Abcam
+              value: abcam
+            - label: BioRad
+              value: biorad
+            - label: ThermoFisher
+              value: thermofisher
+            - label: CellSignal
+              value: cellsignal
+            - label: Diagenode
+              value: diagenode
+            - label: ActiveMotif
+              value: activemotif
+            - label: Other
+              value: other
+        - name: antibody_catalog_number
+          label: Antibody catalog number
+          type: basic:string
+          required: false
+          hidden: "experiment.assay_type != 'chip-seq' && experiment.assay_type != 'chipmentation'"
+        - name: antibody_lot_number
+          label: Antibody lot number
+          type: basic:string
+          required: false
+          hidden: "experiment.assay_type != 'chip-seq' && experiment.assay_type != 'chipmentation'"
```

Comment:
These fields were part of the nested group `chip_seq` before. Should we add some prefix/suffix to the name and label to make it more clear?